src/Sylius/Bundle/ShopBundle/EmailManager/OrderEmailManager.php

```diff
@@ -19,6 +19,7 @@
 use Sylius\Component\Mailer\Sender\SenderInterface;
 use Webmozart\Assert\Assert;
 
+// This whole class needs to go
```

Comment:
What should be the next steps? Did you create a task or smth?

Reply:
https://sylius.atlassian.net/browse/SYL-3109

Reply:
There is a task described, but needs a bit more planning.
I didn't remove the comment since _some_ action needs to be done.